packages/common-utils/src/queryParser.ts

```diff
@@ -299,6 +343,12 @@ export abstract class SQLSerializer implements Serializer {
         `(?? ${isNegatedField ? '!' : ''}= CAST(?, 'Float64'))`,
         [column, term],
       );
+    } else if (propertyType === 'json') {
+      const shoudUseTokenBf = isImplicitField;
```

Comment:
I know it's probably c + p. `shoudUseTokenBf` is misleading here since json type field doesn't have a tokenbf index. we can just do `ILIKE` for now

Reply:
sounds good